Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -390,20 +390,32 @@ def bottle_formula(f)
       mismatches = [:root_url, :prefix, :cellar, :rebuild].reject do |key|
         old_spec.send(key) == bottle.send(key)
       end
-      mismatches.delete(:cellar) if old_spec.cellar == :any && bottle.cellar == :any_skip_relocation
+      if old_spec.cellar == :any && bottle.cellar == :any_skip_relocation
+        mismatches.delete(:cellar)
+        bottle.cellar :any
+      end
```

Comment:
@iMichka Could you please remove remove all diff chunks from this PR except for this one?